lib/adiwg/mdtranslator/readers/iso19115_2/modules/module_temporal_extent.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'adiwg/mdtranslator/internal/internal_metadata_obj'
+require_relative 'module_time_period'
+
+module ADIWG
+   module Mdtranslator
+      module Readers
+         module Iso191152
+            module TemporalExtent
+               @@temporalExtXPath = 'gmd:EX_TemporalExtent'
+               @@timePeriodXPath = './/gml:TimePeriod'
```

Comment:
`extent` is a required property of `EX_TemporalExtent` via 
```xml
<xs:element name="extent" type="gts:TM_Primitive_PropertyType"/>
```
can you add that xpath and grab the element from temporal extent? you'd then pass that element to `TimePeriod`

Reply:
I want to avoid xpaths which search lower than the immediate children of an element. only if it's necessary and makes sense should we search like `.//element` instead of `element`